core/transaction/transaction.go

```diff
@@ -310,24 +310,24 @@ func (tx *Transaction) GetProgramHashes() ([]Uint160, error) {
 		}
 		hashs = append(hashs, astHash)
 	case IssueAsset:
-		result, err := tx.GetTransactionResults()
+		result := tx.GetMergedAssetIDValueFromOutputs()
 		if err != nil {
 			return nil, NewDetailErr(err, ErrNoCode, "[Transaction], GetTransactionResults failed.")
```

Comment:
If there is no err return value from previous function,  why run error condition judge here?